scripts/lotus-daemon.service

```diff
@@ -1,19 +1,17 @@
 [Unit]
 Description=Lotus Daemon
-After=network-online.target
-Requires=network-online.target
+After=network.target
 
 [Service]
-Environment=GOLOG_FILE="/var/log/lotus/daemon.log"
-Environment=GOLOG_LOG_FMT="json"
```

Comment:
Why are we removing these?

Reply:
I suppose this is just personal preference, since systemd already has a mechanism for handling journal output, and this doesn't include a logrotate configuration. I can add this back, though it might eventually fill up disk space with the log file un-handled.

Reply:
Fair enough. You're right, we might as well keep it simple in the default config. The user can always use a drop-in.